docs/source/tutorials/uncertainty.rst

```diff
@@ -0,0 +1,148 @@
+Estimate uncertainty
+=====================
+
+This tutorial shows how to estimate uncertainty measures on the model predictions. Uncertainty measures that are
+implemented in ``ivadomed`` are detailed :doc:`../technical_features#uncertainty-measures`.
+
+.. _Prerequisite:
+
+Prerequisite
+------------
+
+The spinal cord segmentation model generated from :doc:`../tutorials/one_class_segmentation_2d_unet` will be used to
+estimate uncertainty at inference time. Please make sure you did this tutorial prior to this present example and that a
```

Comment:
isn't there a possibility to remove the tutorial dependency to be able to make this tutorial? e.g. using one of the model available in the test data?

Reply:
Great
Issue opened [here](https://github.com/ivadomed/ivadomed/issues/407).